src/Sequence.cpp

```diff
@@ -48,10 +48,10 @@ size_t Sequence::sMaxDiskBlockSize = 1048576;
 
 // Sequence methods
 Sequence::Sequence(
-   const SampleBlockFactoryPtr &pFactory, sampleFormat format)
+   const SampleBlockFactoryPtr &pFactory, SampleFormats formats)
 :  mpFactory(pFactory),
-   mSampleFormat(format),
-   mMinSamples(sMaxDiskBlockSize / SAMPLE_SIZE(mSampleFormat) / 2),
+   mSampleFormats{ formats },
```

Comment:
Do we really need it? It doesn't look readable at all and is not required here

Reply:
Need what?  Uniform initialization?  You don't like it or think it's unreadable?

I now make the habit to use it where I can.  Consistent use of it helps make it clear that questionable implicit conversions aren't happening.

Reply:
Yes, consistency is a problem here. There are cases where uniform initialization is good and welcome. But here, it is inconsistent with the surrounding code and has no special meaning.

Reply:
So I'll take that as an excuse to use uniform initialization on all the members!  Just as I incidentally fix up other things like Doxygenation.